CHANGELOG.md

```diff
@@ -16,6 +45,7 @@
   (PR [#3120](https://github.com/scality/metalk8s/pull/3120))
 
 ## Release 2.7.1
+
```

Comment:
A space were added here and not for `2.7.2` ? :smiley:

Reply:
Removed.